src/plugins/piral-redux/src/create.test.ts

```diff
@@ -1,5 +1,6 @@
-import { Atom, swap } from '@dbeining/react-atom';
+import { Atom, swap, deref } from '@dbeining/react-atom';
 import { createReduxApi } from './create';
+import { MyComponent } from './MyComponent';
```

Comment:
Please use no dedicated file for such things. Define and use such components only in the test files.